packages/web/src/components/cell-output.tsx

```diff
@@ -66,9 +69,9 @@ export function CellStdio({ cell, show, setShow }: PropsType) {
                     'border-transparent data-[state=active]:border-transparent data-[state=active]:text-tertiary-foreground mb-0',
                 )}
               >
-                {tscOutput.length > 0 ? (
+                {diagnostics.length > 0 ? (
```

Comment:
We call this "problems" but right now we're sending all diagnostics to this tab. That includes warnings and suggestions. So... TODO: rework "Problems" tab to be something like "Diagnostics" which has different categories of diagnostics (error, warning, suggestion)